### ISSUE TITLE
setup.py doesn't find all packages

### ISSUE THREAD

**Issue by grifball** [nescio007/teether#1]
Hi, cool project, I'm looking forward to exploring it more.
I just found a problem with "setup.py" early on. It's only finding the teether folder, not sub folders (like teether/cfg). I fixed it by changing this in setup.py:
`
"packages=setuptools.find_packages(),"
`
previously it just had:
`
packages=['teether']
`
it's a quick fix, I could write up a PR if needed.

**Comment by grifball** [comment]
here was the exact issue:
```
$ python3 bin/extract_contract_code.py
Traceback (most recent call last):
  File "bin/extract_contract_code.py", line 5, in <module>
    from teether.cfg.cfg import CFG
  File "/home/vmuser/checkout/teether/venv/lib/python3.6/site-packages/teether-0.1-py3.6.egg/teether/__init__.py", line 1, in <module>
ImportError: cannot import name 'cfg'
```
After a using a new virtual environment and running `python3 setup.py install`

**Comment by nescio007** [comment]
Thanks for reporting, fix should be in commit d7b7fd1
